assets/javascripts/spoiler.js

```diff
@@ -13,12 +13,12 @@
     $("a", $spoiler).addClass("no-track-link");
   }
 
-  var applyBlur = function ($spoiler) {
+  let applyBlur = function ($spoiler) {
     blurLazyYT($spoiler);
     blur($spoiler);
   };
 
-  var applySpoilers = function ($spoiler) {
```

Comment:
Is this asset transpiled? If not, do we want `let` in the output?

Reply:
I don't see this causing any issue anyways no ?

<img width="935" alt="Screenshot 2020-10-13 at 12 31 04" src="https://user-images.githubusercontent.com/339945/95849597-f772b780-0d4f-11eb-9a0a-c635516f987a.png">